pandas/core/frame.py

```diff
@@ -7021,8 +7021,9 @@ def corr(self, method='pearson', min_periods=1):
             * kendall : Kendall Tau correlation coefficient
             * spearman : Spearman rank correlation
             * callable: callable with input two 1d ndarrays
-                and returning a float
-
+                and returning a float. Note that the returned matrix from corr
```

Comment:
I am not sure this is needed to said. This is by-definition that same-same are correlated to 1.0

Reply:
if you want instead to add a link to a correlation matrix that would be ok

Reply:
I wanted to use the method argument of `corr` to compute p-values for the significance of the correlation. Then, the resulting matrix is not a correlation matrix anymore. Currently, pandas assumes that the callable supplied to method computes actual correlations, but this assumption is not explicitly mentioned. At least for me, it would have been helpful to know this behaviour.